web/static/sass/wtt.scss

```diff
@@ -1432,4 +1450,207 @@ h2 {
         box-shadow: 0px 1px 3px rgba(0,0,0,0.2);
         border-radius: 3px;
      }
- }
\ No newline at end of file
+ }
+
+/*
+ * WTT Cards
+ */
+
+.card {
+  background-color: transparentize(#fff, 0.1);
+  padding: 1rem;
+  border-radius: 0.3em;
+
+  > *:first-child {
+    margin-top: 0;
+  }
+
+  > *:last-child {
+    margin-bottom: 0;
+  }
+
+  &-md {
+    padding: 2rem 1rem;
+  }
+  
+  &-lg {
+    padding: 3rem 1rem;
+  }
+
+  &-text-center {
+    text-align: center;
+  }
+  
+  small {
+    font-size: 0.75rem;
```

Comment:
This feels far too small to me for the reassuring part of the page:
![image](https://github.com/user-attachments/assets/39b72cb4-2577-429e-b118-2179d36fbc0b)

I don't think anything here should be under 1rem.